src/core/server/rendering/types.ts

```diff
@@ -74,6 +74,7 @@ export interface RenderingMetadata {
         user: Record<string, UserProvidedValues<any>>;
       };
     };
+    logoURL: string;
```

Comment:
should we be proactive and make this
```
branding: {
  logoURL: string
}
```

then pass the object around

Reply:
NIT: on line 62 of this file I see `translationsUrl` we should match the casing if `Url`